plugins/Morpheus/templates/_iframeBuster.twig

```diff
@@ -1,11 +1,10 @@
 {% if (enableFrames is defined and enableFrames == false) %}
     <script type="text/javascript">
-        $(function () {
-        $('body').css("display", "none");
-        if (self == top) {
-            var theBody = document.getElementsByTagName('body')[0];
-            theBody.style.display = 'block';
-        } else { top.location = self.location; }
-    });
+        if (self !== top) {
+            const theBody = document.getElementsByTagName('body')[0];
```

Comment:
Hm. Not sure if we should use `const` here. I guess it should be valid for most modern browsers that might be used to view Matomo, but using `var` would be more safe though...

Reply:
I checked https://caniuse.com/const vs https://caniuse.com/mdn-javascript_statements_var and based on the percentages decided it was safe to use.
Surely we can go back to `var`, though with that approach we would need to change it in more places, instead of a mix of `var`, `let` and `const`, so I think we can be a tad progressive here.

Reply:
In the past we actually used `var` for all cases that don't run through a compiler. It's safe to use that in all vue code, as it is replaced with `var` when it is compiled.

Reply:
Switched back to `var`.